src/pip/_vendor/tenacity/_utils.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2016 Julien Danjou
+# Copyright 2016 Joshua Harlow
+# Copyright 2013-2014 Ray Holder
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+import sys
+import time
+from functools import update_wrapper
+
+import pip._vendor.six as six
```

Comment:
I had to change this and other lines in the `tenacity` code to adapt to `pip` vendoring system. This causes the `vendoring` tox task to fail on `vendoring run-test: commands[1] | git diff --exit-code`.

Was I not supposed to change the `tenacity` code?